codejail/safe_exec.py

```diff
@@ -36,6 +37,12 @@ def safe_exec(code, globals_dict, files=None, python_path=None, slug=None,
     >>> jail = codejail.jail.get_codejail("python")
     >>> jail.safe_exec(...)
     """
+    if not is_configured("python"):
+        venv = sibling_sandbox_venv()
+        if not venv:
+            raise Exception("Couldn't find sandbox virtualenv")
+        configure("python", os.path.join(venv, "bin/python"), user="sandbox", lang=languages.python2)
```

Comment:
I don't think this is the right place to add this logic.  edx-platform should be deciding where it wants its sandboxes created, not codejail.